lib/IMAP/MessageMapper.php

```diff
@@ -724,7 +724,7 @@ public function getBodyStructureData(Horde_Imap_Client_Socket $client,
 			]);
 			/** @var Horde_Imap_Client_Data_Fetch $part */
 			$part = $parts[$fetchData->getUid()];
-			$htmlBody = $part->getBodyPart($htmlBodyId);
+			$htmlBody = ($htmlBodyId !== null) ? $part->getBodyPart($htmlBodyId) : [];
```

Comment:
Why an empty array and not an empty string or null?

Reply:
Good question
![image](https://user-images.githubusercontent.com/7427347/203778508-4c926966-26ae-43ab-ba71-3062b8af3a52.png)